trident/absorption_spectrum/absorption_line.py

```diff
@@ -220,17 +235,24 @@ def tau_profile(lambda_0, f_value, gamma, v_doppler, column_density,
     # dimensionless frequency offset in units of doppler freq
     x = _cs / v_doppler * (lam1 / lambda_bins - 1.0)
     a = gamma / (4.0 * np.pi * nudop)               # damping parameter
-    phi = voigt(a, x)                               # line profile
+    if deposition_method == 'voigt':
+        phi = voigt(a, x)                               # line profile
+    else:
+        phi = delta(lam1,lambda_bins)
     tauphi = tau0 * phi              # profile scaled with tau0
 
     return (lambda_bins, tauphi)
 
 def tau_profile_21cm(lambda_0, f_value, gamma, temperature, number_density,
```

Comment:
I don't think we need both a `tau_profile_21cm` function and a variable deposition method in the regular `tau_profile` function. I would say choose one or the other, perhaps whichever option has the least amount of duplicate code.

Reply:
Agreed.  I would say we could just do with `tau_profile` and have a keyword which triggers the 21 cm deposition to be different, and in turn, this would get triggered by using a line like: `H I 21cm` or something?